packages/react-scripts/template/gitignore

```diff
@@ -11,8 +11,12 @@
 # production
 /build
 
+# development
+/dist
+
 # misc
 .DS_Store
+.env
 .env.local
```

Comment:
do we have some sort of pattern for alternate .env files? Should we just do `.env.*`? Or does something generate files like this, and if so, why?